src/main/java/org/example/Main.java

```diff
@@ -1,7 +1,220 @@
 package org.example;
 
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
 public class Main {
+    private static final Scanner scanner = new Scanner(System.in);
+    private static final ArrayList<Task> tasks = new ArrayList<>();
+    private static final String SAVE_FILE = "saved_tasks.csv";
+    private static boolean running = true;
+
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        String[] allowedCommands = {"add", "delete", "list", "info", "save", "view", "update"};
+
+        Main.info(allowedCommands);
```

Comment:
CLEAN-CODE: zvolil bych vice popisnou metodu.

```suggestion
        displayAllowedCommands();
```